packages/lesswrong/lib/utils/schemaUtils.ts

```diff
@@ -133,6 +145,16 @@ export const foreignKeyField = <CollectionName extends CollectionNameString>({id
         fieldName: idFieldName,
         nullable,
       }),
+      ...(autoJoin ? {
+        sqlResolver: ({field, join}: SqlResolverArgs) => join({
+          table: collectionName,
+          type: nullable ? "left" : "inner",
+          on: {
+            _id: field(idFieldName),
+          },
+          resolver: (foreignField) => foreignField("*"),
```

Comment:
I think this should include an accessFilterSingle check somehow, because `generateIdResolverSingle` does